Make maven build to use package lock

When naively trying to change to do `npm ci`:
```
$ git diff
diff --git a/pom.xml b/pom.xml
index c7f05dc7..aad49336 100644
--- a/pom.xml
+++ b/pom.xml
@@ -111,7 +111,7 @@
                                                        <goal>npm</goal>
                                                </goals>
                                                <configuration>
-                                                       <arguments>install --scripts-prepend-node-path=auto</arguments>
+                                                       <arguments>ci</arguments>
                                                </configuration>
                                        </execution>
                                        <execution>

```

I get:

```
$ ./mvnw clean package
...
[INFO] --- frontend-maven-plugin:1.12.0:npm (npm install) @ spring-cloud-dataflow-ui ---
[INFO] Running 'npm ci' in /home/jvalkealahti/repos/jvalkeal/spring-cloud-dataflow-ui/ui
[INFO] npm ERR! cipm can only install packages with an existing package-lock.json or npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or later to generate it, then try again.
[INFO] 
[INFO] npm ERR! A complete log of this run can be found in:
[INFO] npm ERR!     /home/jvalkealahti/.npm/_logs/2021-10-29T14_31_48_882Z-debug.log
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------

```

--- a/pom.xml
+++ b/pom.xml
@@ -156,13 +156,6 @@
 							<followSymlinks>false</followSymlinks>
 						</fileset>
 						<fileset>
-							<directory>ui</directory>
-							<includes>
-								<include>package-lock.json</include>
-							</includes>
-							<followSymlinks>false</followSymlinks>
-						</fileset>
-						<fileset>
 							<directory>ui/dist</directory>
 							<followSymlinks>false</followSymlinks>
 						</fileset>

